prow/github/fakegithub/fakegithub.go

```diff
@@ -195,3 +195,7 @@ func (f *FakeClient) GetFile(org, repo, file, commit string) ([]byte, error) {
 
 	return nil, fmt.Errorf("could not find file %s with ref %s", file, commit)
 }
+
+func LabelID(org, repo string, number int, label string) string {
```

Comment:
i noticed that github/ has a FakeClient method but it seemed to be added before the fakegithub/ dir so I figured current practice was to use the fakegithub client.

I can avoid using fakegithub by creating my own mock in the test.